code/game/machinery/mass_driver.dm

```diff
@@ -14,10 +14,6 @@
 	. = ..()
 	wires = new /datum/wires/mass_driver(src)
 
-/obj/machinery/mass_driver/Destroy()
-	QDEL_NULL(wires)
```

Comment:
this isn't a side effect it's garbage collection, wires retain a reference to us if we leave them hanging we hard delete.

Reply:
`QDEL_NULL(wires)` is still there in line 34. Mass Drivers had two Destroy defines, I merged the two.

Reply:
aight, copy that